Dockerfile

```diff
@@ -10,6 +10,11 @@ RUN apk add libusb-dev linux-headers
 WORKDIR /code
 COPY . /code
 
+# try this one out
+RUN BUILD_TAGS=muslc make view
+
+RUN false
+
```

Comment:
This diff should probably not be here, right?

Reply:
Totally... debug cleanup :boom: